extmod/modlwip.c

```diff
@@ -1620,13 +1620,18 @@ STATIC MP_DEFINE_CONST_OBJ_TYPE(
 // Support functions for memory protection. lwIP has its own memory management
 // routines for its internal structures, and since they might be called in
 // interrupt handlers, they need some protection.
+
+// Added protection flag. Only few ports are providing and OS
```

Comment:
I think the comment should be rephrased.

Reply:
done